### PR TITLE
Accept rubocop disable flag

### PR DESCRIPTION
**Pull request by danielmarzini**
The change allow to exclude a file from the analysis if it has the comment
```# rubocop disable all```

The comment should be placed on the first line of the file.

Such feature is critical when ruumba is integrated on a repo with a big files base, since it allows to enable ruumba on a CI and softly rollout the analysis.

### CODE REVIEW

File: lib/ruumba/analyzer.rb [hunk 1]
```diff
@@ -46,7 +46,10 @@ def analyze(temp_dir, files_or_dirs)
         end
 
       iterator.each do |file, contents|
-        copy_erb_file(file, contents, temp_dir_path)
+        fileFirstLine = File.open(file, &:readline)
```
Comment (ericqweinstein) [inline lib/ruumba/analyzer.rb]: @danielmarzini Thanks for your contribution! This looks reasonable to me. A couple of notes:

1. Prefer `file_first_line` to `fileFirstLine`
2. Prefer `unless` to `if !` here
3. Prefer `'` to `"` in the absence of interpolation
4. Prefer two spaces for indentation

After that, I think we're mergeable 👍

Comment (danielmarzini) [reply lib/ruumba/analyzer.rb]: tnx, commit on its way.

**Comment by danielmarzini** [comment]
@ericqweinstein updated as requested.

**Comment by ericqweinstein** [comment]
@danielmarzini Thanks for this! I'll get CI green tonight (it's probably mad about the `Gemfile.lock`) and merge/deploy once we're in good shape.

**Comment by ericqweinstein** [comment]
Yeah, I've just gotta rebundle. I'll fix merge, fix, then cut a new release.

**Comment by ericqweinstein** [comment]
Hmm, looks like a bunch of path-related build failures—shouldn't be related, but I'll dig in and fix as soon as I can. I'll cut a new tag once the build is green again and then release that version.

**Comment by aclemons** [comment]
Just wonder why this change was necessary? Since b9c678c ruumba preserves comments so you should be able to just add:

`<%# rubocop:disable all %>`

to any ERB file and the standard rubocop behaviour will be used. Did this not work as expected?

**Comment by aclemons** [comment]
The specs are broken because `spec/ruumba/analyzer_spec.rb` was not changed to accommodate `lib/ruumba/analyzer.rb` calling `File.open` to check for this comment.

**Comment by ericqweinstein** [comment]
@aclemons @danielmarzini Sorry, I was sick for a bit. I'll ship the above fix over the weekend at the latest. My understanding was that the comment wasn't being parsed correctly—I'll confirm ASAP (though if either of you could take a look and let me know, it would be much appreciated). If the magic comment works without this modification, I'll go ahead and revert.

Thanks again for all your hard work!

**Comment by ericqweinstein** [comment]
Ah, never mind—see #36 for details.

**Pull request merged**